src/views/AddPeopleAndRoles.vue

```diff
@@ -35,20 +35,8 @@ export default class AddPeopleAndRoles extends Mixins(EntityFilterMixin) {
   // Global getters
   @Getter isEntityType!: GetterIF
 
-  // Global actions
-  @Action setIgnoreChanges!: ActionBindingIF
-
-  // Entity Enum
+  // Enum for template
   readonly EntityTypes = EntityTypes
-
-  /** Called when component is created. */
-  private created (): void {
-    // ignore data changes until page has loaded
-    this.setIgnoreChanges(true)
```

Comment:
The idea here is that you disable change tracking while the component loads its data (and re-enable after). That way, just loading the page doesn't display the "you have unsaved changed" dialog when you reload/close the app.
Unfortunately it is not used consistently in Create UI (for which I created a tech debt ticket). Let's use it for our new views as intended 👍

Reply:
Sounds good to me!